src/main/resources/db/changelog/changes/06-insert-users.yaml

```diff
@@ -0,0 +1,41 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert-users
+      author: samoilenko
+      changes:
+        - insert:
+            tableName: users
+            columns:
+              - column:
+                  name: email
+                  value: "admin@mail.com"
+              - column:
+                  name: password
+                  value: "$2a$10$OrfBaEepFwU.OKOH/KMJ0.1IwfZVX5AnMp2iUsXxANhTIJK58ZCAO"
+              - column:
+                  name: first_name
+                  value: "Admin"
+              - column:
+                  name: last_name
+                  value: "Admin"
+              - column:
+                  name: shipping_address
+                  value: "Adminovsk"
```

Comment:
this field is actually optional, and can be left with null value, but I like this Adminovsk more 😄